filemanipulation_blueprints/compress_file.py

```diff
@@ -97,28 +97,28 @@ def compress_files(file_paths, destination_full_path, compression):
     if compression == 'zip':
         with ZipFile(compressed_file_name, 'w') as zip:
             for file in file_paths:
-                file = file.replace(os.getcwd(), '')
+                file = file.replace(os.getcwd(), '.')
```

Comment:
I believe using pathlib here would be more a more robust solution but this blueprint is also simple enough that we may be over optimizing it by using pathlib.

if you have the values that are currently breaking this I would test it against using 
``` 
from pathlib import Path
...
file = Path(file.relative_to(Path.cwd())
```